time_series/time_series_aggregations.qmd

```diff
@@ -348,65 +317,63 @@ Why didn't we list with t.rast.list?
 
 ### Bioclimatic variables
 
-Perhaps you have heard of [Worldclim](https://www.worldclim.org/) or 
-[CHELSA](https://chelsa-climate.org/) bioclimatic variables? Well, these are 19 
-variables that represent potentially limiting conditions for species. They 
-derive from the combination of temperature and precipitation long term 
-aggregations. 
+Perhaps you have heard of [Worldclim](https://www.worldclim.org/) or
+[CHELSA](https://chelsa-climate.org/) bioclimatic variables? Well, these are 19
+variables that represent potentially limiting conditions for species. They
+derive from the combination of temperature and precipitation long term
+aggregations.
 Let's use those that we estimated in the previous example to estimate the
 bioclimatic variables that include temperature. GRASS GIS has a very nice
-extension, 
+extension,
 [r.bioclim](https://grass.osgeo.org/grass-stable/manuals/addons/r.bioclim.html),
 to estimate bioclimatic variables.
 
 ```{python}
 # Install extension
-gs.run_command("g.extension",
-               extension="r.bioclim")
+gs.run_command("g.extension", extension="r.bioclim")
 ```
 
 ```{python}
 # Get lists of maps needed
-tmin=gs.list_grouped(type="raster", pattern="lst_minimum_??")["italy_LST_daily"]
-tmax=gs.list_grouped(type="raster", pattern="lst_maximum_??")["italy_LST_daily"]
-tavg=gs.list_grouped(type="raster", pattern="lst_average_??")["italy_LST_daily"]
+tmin = gs.list_grouped(type="raster", pattern="lst_minimum_*")["italy_LST_daily"]
+tmax = gs.list_grouped(type="raster", pattern="lst_maximum_*")["italy_LST_daily"]
+tavg = gs.list_grouped(type="raster", pattern="lst_average_*")["italy_LST_daily"]
```

Comment:
I needed numbers, that's why I used the `??`. Does it do the right selection with the *?